instrumentation/opentelemetry-instrumentation-threading/README.rst

```diff
@@ -0,0 +1,23 @@
+OpenTelemetry Threading Tracing
+===========================
+
+|pypi|
+
+# To-Do : Whats is the image for opentelemetry-instrumentation-threading?
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-instrumentation-flask.svg
```

Comment:
@srikanthccv What should this image be for the threading module?

Reply:
Just use `opentelemetry-instrumentation-threading.svg`